compiler/ballerina-lang/src/main/java/org/wso2/ballerinalang/compiler/semantics/analyzer/ConstantValueResolver.java

```diff
@@ -596,7 +607,7 @@ private void checkUniqueness(BLangConstant constant) {
                     dlog.error(constant.name.pos, DiagnosticErrorCode.ALREADY_INITIALIZED_SYMBOL, nameString);
                 } else {
                     BLangConstantValue lastValue = constantMap.get(nameString);
-                    if (!value.equals(lastValue)) {
+                    if (!value.toString().equals(lastValue.toString())) {
```

Comment:
Made this change to fix CCE for 
```
const NUM1 = -1;
const int NUM2 = -9223372036854775807 - 1;
const int ANS7 = NUM2 - 1; 
const int ANS7 = NUM2 + NUM1; 
```

Is that acceptable?

Reply:
This change causes to allow invalid constant declarations like following example.

```ballerina
public enum LiftStatus {
    CLOSED = "123"
}

const CLOSED = 123; // No errors!
```

Reply:
Updated b3fb7aaad3e156c54a1388007bd0154774194e42